@remirror/extension-code-block/src/code-block-extension.ts

```diff
@@ -181,7 +181,7 @@ export class CodeBlockExtension extends NodeExtension<CodeBlockExtensionOptions>
    * Create an input rule that listens converts the code fence into a code block with space.
    */
   public inputRules({ type }: ExtensionManagerNodeTypeParams) {
-    const regexp = /^```([a-zA-Z]*)? $/;
```

Comment:
`'``` '.match(/^```([a-zA-Z]*)? $/)` will return `{0: '``` ', 1: undefined, index: 0, input: '``` ', groups: undefined}`, which means the `language` variable is `undefined`. This could break something since the TypeScript assume that `language` is a `string`.